### ISSUE TITLE
when adding encryption or other submission element features, remove error

### ISSUE THREAD

**Issue by MartijnR** [enketo/enketo-transformer#65]
"Submissions element(s) not supported."

**Comment by lognaturel** [comment]
I suspect that this issue precedes the Enketo implementation of encryption. Should be verified.

**Comment by MartijnR** [comment]
Yes, indeed. It's just this about removing this cod,e I think: https://github.com/enketo/enketo-transformer/blob/master/src/xsl/openrosa2html5form.xsl#L224-L226. The error is logged but not shown to the user.
